environment-dev.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - openmm
   - parmed>=3.4.3
   - ele
-  - openff-toolkit >=0.11
+  - openff-toolkit-base >=0.11
```

Comment:
Does this just install a lighter version of openff-toolkit?

Reply:
precisely - and some of the heavier ones/those that are more likely to cause conflicts

https://github.com/conda-forge/openff-toolkit-feedstock/blob/f8d07c4999274f3566062501038537c9ddf1569f/recipe/meta.yaml#L63-L66

Foyer only does some basic interoperability so stuff like partial charge assignment isn't needed. `Molecule.from_smiles` requires RDKit, but that's brought in by mBuild so it worked out here

Reply:
Ohhh ok, that's cool! I didn't know you could create multiple package options like that in a single meta.yml file. I'll have to remember that.